Libraries/Image/RCTImageView.m

```diff
@@ -96,10 +96,10 @@ - (void)updateImage
   }
 
   // Applying capInsets of 0 will switch the "resizingMode" of the image to "tile" which is undesired
```

Comment:
Can you update this comment? I'd also make the resizeMode check the first part of the if.